packages/components/tests/dummy/app/styles/showcase-components/placeholder.scss

```diff
@@ -18,6 +18,7 @@
   line-height: 1.2;
   text-align: center;
   text-shadow: 0 0 5px #fff;
+  background-color: #eee;
```

Comment:
Thanks for adding this. It makes the placeholder easier to use besides just fixing the contrast issue.

Reply:
I am not sure why I didn't think of it before 🤷‍♂️🤦‍♂️